bin/kn

```diff
@@ -103,7 +114,8 @@ function validate_pwd() {
 
 # Set defaults if not specified in the environment
 if [ -z "$KN_PROVISIONERS_IMG" ]; then
-  KN_PROVISIONERS_IMG="kubenow/provisioners:$kubenow_version"
+  tag="${kubenow_version/\//-}"
+  export KN_PROVISIONERS_IMG="kubenow/provisioners:$tag"
```

Comment:
Part of the possibility to have different provisioners image depenting on if you are on a development branch.